def/conference.go

```diff
@@ -1,7 +1,20 @@
 package def
 
-// Conference is an event
-// gorm: "true"
+import "time"
+
+// Conference is a brand like GopherCon
 type Conference struct {
-	Name string
+	ID     uint
+	Name   string
+	Events []Event
+}
+
+// Event is an instance like GopherCon 2020
+type Event struct {
```

Comment:
A tagline sounds like something we could use? If memory serves gophercons tend to have one, such as `#GoVirCon ~ see you online!`